Edi/Edi.Apps/ViewModels/ApplicationViewModel_Config.cs

```diff
@@ -88,16 +84,16 @@ public void OnClosing(object sender, System.ComponentModel.CancelEventArgs e)
         {
             try
             {
-                if (this.Exit_CheckConditions(sender) == true)      // Close all open files and check whether application is ready to close
```

Comment:
This change is not OK.
I prefer the more expressive if clause using a comparison with true or false instead of using the not operator '!' or its absence.